components/emoji_picker/emoji_picker.tsx

```diff
@@ -105,8 +109,15 @@ const EmojiPicker = ({
 
         setCategoryOrEmojisRows(updatedCategoryOrEmojisRows);
         setEmojiPositionsArray(updatedEmojiPositions);
+        throttledSearchCustomEmoji.current(filter, customEmojisEnabled);
     }, [filter, userSkinTone, shouldRunCreateCategoryAndEmojiRows.current]);
 
+    const throttledSearchCustomEmoji = useRef(throttle((newFilter, customEmojisEnabled) => {
```

Comment:
Could you move this up to be defined with the other `useRef`s above? We don't have a good style guide for this yet, but I think we'd generally want to keep the usage of each type of hook together

Reply:
Makes sense. Done